### PR TITLE
[extractor/twitcasting] Fix extractor for protected twitcast videos

### LINKED ISSUE #5888: [Twitcasting] Download failing on protected videos
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

_No response_

### Provide a description that is worded well enough to be understood

There is an issue trying to download Twitcast videos and more specifically password protected videos. The issue may appear to be an extractor error when searching through the request form.  Protected videos can not be downloaded when providing a password nor will unlocking the videos manually and using the cookies from the browser work anymore.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--cookies-from-browser', 'chrome', '--video-password', 'PRIVATE', '-o', 'test.mkv', 'https://twitcasting.tv/yuzukichococh/movie/661534618']
[debug] Encodings: locale cp65001, fs utf-8, pref cp65001, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (pip)
[debug] Python 3.11.0 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1q  5 Jul 2022)
[debug] exe versions: ffmpeg N-107100-g6a7c6ee93f-g4d45f5acbd (setts), ffprobe N-107100-g6a7c6ee93f-g4d45f5acbd
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-9.1
[Cookies] Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\test\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\test\AppData\Local\Google\Chrome\User Data\Local State"
[Cookies] Extracted 2285 cookies from chrome
[debug] cookie version breakdown: {'v10': 2285, 'other': 0, 'unencrypted': 0}
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [TwitCasting] Extracting URL: https://twitcasting.tv/yuzukichococh/movie/661534618
[TwitCasting] 661534618: Downloading webpage
ERROR: [TwitCasting] 661534618: This video is protected by a password, use the --video-password option
  File "C:\Users\test\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\extractor\common.py", line 674, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\test\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\extractor\twitcasting.py", line 107, in _real_extract
    raise ExtractorError('This video is protected by a password, use the --video-password option', expected=True)
```

### PR DESCRIPTION
**Pull request by Spicadox**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

Apply patch provided by @bashonly  to fix twitcasting extractor on password protected videos.

Fixes #5888 

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

https://github.com/yt-dlp/yt-dlp/issues/5888#issuecomment-1366311045

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

**Comment by pukkandan** [comment]
this is @bashonly's patch, correct?

**Comment by Spicadox** [comment]
> this is @bashonly's patch, correct?

Yep, correct

File: yt_dlp/extractor/twitcasting.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class TwitCastingIE(InfoExtractor):
             'description': 'Twitter Oficial da cantora brasileira Ivete Sangalo.',
             'thumbnail': r're:^https?://.*\.jpg$',
             'upload_date': '20110822',
-            'timestamp': 1314010824,
+            'timestamp': 1313978424,
```
Comment (pukkandan) [inline yt_dlp/extractor/twitcasting.py:41]: How come timestamp changed?

Comment (bashonly) [reply yt_dlp/extractor/twitcasting.py:41]: Not sure. It looks like the webpage format had changed slightly, I also had to fix the `view_count` extraction

Comment (Lesmiscore) [reply yt_dlp/extractor/twitcasting.py:41]: image
image
Looks like they seem to store the timestamp by UTC now? (the difference is exactly 9 hours)

**Pull request merged**